src/app/account-select/account-select.component.html

```diff
@@ -1,42 +1,37 @@
-<div class="d-flex flex-column" *ngIf="hasUsers">
-  <ul class="list-group mt-7px mb-30px saved-seeds-list">
-    <span class="saved-seeds-header d-flex align-items-center"
-      ><span>{{componentTitle}}</span></span
+<div class="section--accounts" *ngIf="hasUsers">
+  <div class="section--accounts__header margin-bottom--medium">
```

Comment:
We probably want a little spacing between component title and the user selection.